flink-staging/flink-streaming/flink-streaming-connectors/flink-connector-rabbitmq/src/main/java/org/apache/flink/streaming/connectors/rabbitmq/RMQSource.java

```diff
@@ -86,7 +85,7 @@ public void close() throws Exception {
 	}
 
 	@Override
-	public void run(Object checkpointLock, Collector<OUT> out) throws Exception {
+	public void run(SourceContext<OUT> ctx) throws Exception {
 		while (running) {
 			delivery = consumer.nextDelivery();

```

Comment:
Make delivery a stack reference. No need to hold a reference in a field longer than necessary.